xrpl/wallet/main.py

```diff
@@ -77,8 +112,88 @@ def create(
         Returns:
             The wallet that is generated from the given seed.
         """
-        seed = generate_seed(algorithm=crypto_algorithm)
-        return cls(seed, sequence=0, algorithm=crypto_algorithm)
+        return cls(algorithm=crypto_algorithm)
+
+    @classmethod
+    def from_public_private_keys(
+        cls: Type[Wallet],
+        public_key: str,
+        private_key: str,
+        master_address: Optional[str] = None,
+    ) -> Wallet:
+        """
+        Generates a new Wallet from public and private keys.
+
+        Args:
+            public_key: The public key for the account.
+            private_key: The private key used for signing transactions for the account.
+            master_address: Include if a Wallet uses a Regular Key Pair. It must be
+                the master address of the account. The default is `None`.
+
+        Returns:
+            The wallet that is generated from the given keys.
+        """
+        wallet = cls()
+        wallet.seed = None
+        wallet.public_key = public_key
+        wallet.private_key = private_key
+        wallet.classic_address = (
+            ensure_classic_address(master_address)
+            if master_address is not None
+            else derive_classic_address(public_key)
+        )
+        return wallet
+
+    @classmethod
+    def from_seed(
```

Comment:
This feels unnecessary given the constructor

Reply:
Yeah I agree. Wanted to make it similar to xrpl.js but not sure if it matters too much. Should I remove from_seed and from_secret as a whole?